### ISSUE TITLE
Customized connection pool of redis cluster

### ISSUE THREAD

**Issue by cque7** [redis/redis-py#2350]
Hello,
In cluster mode, does redis-py support customized connection pool as redis-py-cluster?

Thanks.

**Comment by juanocana** [comment]
hi, 
maybe i'm wrong, but i was not able to use a BlockingConnectionPool on cluster mode, and i guess the main reason come from  this method:

```python
# cluster.py
def create_redis_node(self, host, port, **kwargs):
        if self.from_url:
            # Create a redis node with a costumed connection pool
            kwargs.update({"host": host})
            kwargs.update({"port": port})
            r = Redis(connection_pool=ConnectionPool(**kwargs))
        else:
            r = Redis(host=host, port=port, **kwargs)
        return r
```

**Comment by zach-iee** [comment]
+1
i'm also in need of this feature 😢

**Comment by zakaf** [comment]
+1
i'm also in need of this feature 😢

**Comment by IdoKendo** [comment]
I see that this was merged in #2547 - But the support is still missing in the asyncio client, is there a reason or just overlooked? I can create the PR for the asyncio client if it's fine.

**Comment by zakaf** [comment]
@IdoKendo i personally wasn't using asyncio client, so i didn't add support there.
From my experience, if you make a PR for features, maintainers would be grateful for your effort
